README.md

```diff
@@ -21,10 +21,10 @@ A library for Simple & Efficient data access in Scala and Scala.js
 
 Add the following dependency to your project's build file.
 
-For Scala 2.11.x and 2.12.x:
+For Scala 2.12.x through 3.x:
 
 ```scala
-"com.47deg" %% "fetch" % "2.1.1"
+"com.47deg" %% "fetch" % "3.0.0"
 ```
 
 Or, if using Scala.js (0.6.x):
```

Comment:
We would need to update the version here too.

Reply:
Specifically, the Scala.js version is now 1.x (I assume) and the fetch version is 3.0.0

Reply:
It's technically 1.8.x as per the compatibility rules here: https://www.scala-js.org/news/2021/12/10/announcing-scalajs-1.8.0/

> It is not forward binary compatible with 1.7.x: libraries compiled with 1.8.0 cannot be used with 1.7.x or earlier.

So I've adjusted it to match.